docs/examples/dynamics/controlled_integrator2.ipynb

```diff
@@ -5,7 +5,20 @@
    "metadata": {},
    "source": [
     "# Controlled integrator 2\n",
-    "This demo implements a controlled one-dimensional neural integrator that is functionally the same as the controlled integrator in the previous example. However, the control signal is zero for integration, less than one for low-pass filtering, and greater than 1 for saturation. This behavior maps more directly to the differential equation used to describe an integrator: $\\dot{x} = \\mathrm{Ax}(t) + \\mathrm{Bu}(t)$ The control in this circuit is A in that equation. This is also the controlled integrator described in the book “How to build a brain.”\n"
+    "\n",
+    "This demo implements a controlled one-dimensional neural integrator\n",
+    "that is functionally the same as\n",
+    "the controlled integrator in the previous example.\n",
+    "However, the control signal is zero for integration,\n",
+    "less than one for low-pass filtering, and greater than 1 for saturation.\n",
+    "This behavior maps more directly to the differential equation\n",
+    "used to describe an integrator:\n",
+    "\n",
+    "$$\\dot{x} = \\mathrm{Ax}(t) + \\mathrm{Bu}(t)$$\n",
+    "\n",
+    "The control in this circuit is $A$ in that equation.\n",
+    "This is also the controlled integrator\n",
+    "described in the book \"How to build a brain.\""
```

Comment:
I believe the puncutation should be outside of the quotes of the title (if the English rules don't differ from the German rules).

Reply:
English often puts them inside, even when it doesn't make sense IMO. I don't think it's worth changing.